src/elements/Header/_Header.js

```diff
@@ -1,17 +1,22 @@
-import _ from 'lodash'
-import React, { Component, PropTypes } from 'react'
-import classNames from 'classnames'
+import cx from 'classnames'
+import React, { createElement, Component, PropTypes } from 'react'
+
 import META from '../../utils/Meta'
-import getUnhandledProps from '../../utils/getUnhandledProps'
 
 export default class _Header extends Component {
   static propTypes = {
     _headerElement: PropTypes.string,
+    _sdClass: PropTypes.string,
     children: PropTypes.node,
     className: PropTypes.string,
     icon: PropTypes.node,
     image: PropTypes.node,
-  };
+  }
```

Comment:
Babel reverting their ridiculous breaking change where they threw errors if you didn't have them ;)

I actually meant to leave all them in and i'll pull them on another PR.  Will leave this here for now...